python/cudaq/runtime/observe.py

```diff
@@ -123,6 +125,11 @@ def observe(kernel,
     else:
         ctx = cudaq_runtime.ExecutionContext('observe', shots_count)
         ctx.setSpinOperator(localOp)
+        if num_trajectories != None:
```

Comment:
```suggestion
        if num_trajectories is not None:
```
not sure about style here, but the next line uses `is` so...